segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorageMockTests.java

```diff
@@ -518,9 +515,6 @@ public void testReport() throws Exception {
 
         // Not possible to mock any other reporter except metadata store.
         verify(spyMetadataStore).report();
-
-        Assert.assertEquals(123, MetricRegistryUtils.getGauge(SLTS_STORAGE_USED_BYTES).value(), 0);
```

Comment:
what error does it run into if this is not removed? Would it help if you try and retrieve the specific gauge with the tag passed, in case this gauge is generated with tag.